src/app/QuickStarts/quickstarts/topology/custom-target-quickstart.tsx

```diff
@@ -53,17 +53,17 @@ However, in some cases (e.g. for **Kubernetes API**, JMX port is not <code>9091<
         },
       },
       {
-        title: 'Open Custom Target Creation Form',
+        title: 'Open Custom Target creation form',
```

Comment:
Now that I'm looking at this one, it makes sense that `Custom Target` is capitalized since those are a sort of custom resource and one that the user can create/delete. But in that case, maybe we should keep `Target JVM` instead of replacing it with `target JVM` elsewhere? The `Target` is a sort-of-resource entity, similar to the `Match Expression`.

Reply:
Yes, makes sense.